src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -59,6 +59,31 @@ public function __construct(ContainerInterface $container = null, bool $debug =
         $this->sessionOptions = $sessionOptions;
     }
 
+    private function getSessionOptions(): array
+    {
+        return $this->mergePhpSessionOptions($this->sessionOptions);
+    }
+
+    private function mergePhpSessionOptions(array $sessionOptions): array
```

Comment:
no need for this method, adding getSessionOptions is enough to me

Reply:
done